ui/src/app/edge/live/common/grid/flat/flat.html

```diff
@@ -1,8 +1,8 @@
 <oe-flat-widget [icon]="{name: 'oe-grid', color:'dark'}" [title]="'General.grid' | translate" (click)="presentModal()">
-    <oe-flat-widget-line [name]="'General.gridBuyAdvanced'| translate" [value]="gridBuyPower"
+    <oe-flat-widget-line [name]="'General.gridSellAdvanced' | translate" [value]="gridSellPower"
         [converter]="CONVERT_WATT_TO_KILOWATT">
     </oe-flat-widget-line>
-    <oe-flat-widget-line [name]="'General.gridSellAdvanced' | translate" [value]="gridSellPower"
+    <oe-flat-widget-line [name]="'General.gridBuyAdvanced'| translate" [value]="gridBuyPower"
```

Comment:
Why did you swap the lines for GridBuy and GridSell?

Order should be the same in Live + History:
![grafik](https://github.com/OpenEMS/openems/assets/3515268/7a6710c7-855a-4047-884c-0d851f504269)

Reply:
i changed these because i talked to @ludwig.asen and he suggested that the order in energymonitor and everywhere else should be the same and he also approved this

Reply:
do you still need these pictures?

Reply:
Please add this info to the PR/commit message. This new order should be communicated to users.